test/functional/fixtures/api/es-next/auth/testcafe-fixtures/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Maybe we can place this file at the `fixture` directory to avoid duplicating it everywhere?

Reply:
I've already implemented another solution.